business-central-tests/pom.xml

```diff
@@ -290,7 +292,7 @@
                 <configuration>
                   <properties>
                     <cargo.jboss.configuration>standalone</cargo.jboss.configuration>
-                    <cargo.jvmargs>-Xmx1024m</cargo.jvmargs>
+                    <cargo.jvmargs>-Xmx1024m -Dorg.uberfire.nio.git.daemon.enabled=true</cargo.jvmargs>
```

Comment:
for an unknown reason, it only works if I set it here, the above systemProperties doesn't work